packages/app/src/lib/Camera2D.tsx

```diff
@@ -1,7 +1,8 @@
 import { createMemo } from 'solid-js'
 import { tgpu } from 'typegpu'
 import { f32, mat3x3f, mat4x4f, struct, vec2f, vec3f } from 'typegpu/data'
-import { mat3, mat4, vec3 } from 'wgpu-matrix'
+import { mul } from 'typegpu/std'
+import { mat3, mat4 } from 'wgpu-matrix'
```

Comment:
this is the only usage of `wgpu-matrix` left. We should get rid of it because it is large and doesn't work very well anyways.

Reply:
So we can just write small util to calculate this, is this matrix wgpu performant or was it not right with it?